test/test-batch-transfer.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+# This is a sample Git LFS test.  See test/README.md and testhelpers.sh for
+# more documentation.
+
+. "test/testlib.sh"
+
+begin_test "batch transfer"
+(
+  set -e
+
+  # This initializes a new bare git repository in test/remote.
+  # These remote repositories are global to every test, so keep the names
+  # unique.
+  reponame="$(basename "$0" ".sh")"
+  setup_remote_repo "$reponame"
+
+  # Clone the repository from the test Git server.  This is empty, and will be
+  # used to test a "git pull" below. The repo is cloned to $TRASHDIR/clone
+  clone_repo "$reponame" clone
+
+  # Clone the repository again to $TRASHDIR/repo. This will be used to commit
+  # and push objects.
+  clone_repo "$reponame" repo
+
+  # This executes Git LFS from the local repo that was just cloned.
+  out=$($GITLFS track "*.dat" 2>&1)
+  echo "$out" | grep "Tracking \*.dat"
+
+  contents=$(printf "a")
```

Comment:
Ha, you got this from my `test-happy-path.sh`. Unsure why `contents="a"` wouldn't work :)